substrate/cli/src/cli.yml

```diff
@@ -46,6 +46,14 @@ args:
       value_name: PORT
       help: Specify p2p protocol TCP port
       takes_value: true
+  - rpc-external:
+      long: rpc-external
+      help: Listen to all rpc interfaces (Default is local)
+      takes_value: false
+  - ws-external:
```

Comment:
I believe there is no point in running `RPC` externally but not `WS`, would merge the two flags to avoid clutter in CLI help.
In the future we will most likely run HTTP & WS transport on the same port anyway, so maybe it will be a good opportunity to merge the two options as well.

So I guees it's all right :)

Reply:
I did not want to be too pushy here and let for now the users decide. That allows at the moment a user to 'open up' `WS` while keeping `RPC` only local only. It may be better security-wise for now.